rpcserver.go

```diff
@@ -6808,6 +6841,15 @@ func (r *rpcServer) ForwardingHistory(ctx context.Context,
 			AmtInMsat:   uint64(amtInMsat),
 			AmtOutMsat:  uint64(amtOutMsat),
 		}
+
+		if !req.SkipPeerAliasLookup {
+			resp.ForwardingEvents[i].PeerAliasIn = getRemoteAlias(
```

Comment:
Can we add a small check that the alias is set in https://github.com/lightningnetwork/lnd/blob/master/lntest/itest/lnd_multi-hop-payments_test.go#L276? No need to write a completely new integration test.

Reply:
Thanks for pointing that out, I added checks.